package.json

```diff
@@ -4,7 +4,10 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "compile:sass": "node-sass sass/main.scss css/style.css -w"
+    "compile:sass": "node-sass sass/main.scss css/style.css -w",
+    "json:server": "json-server api/db.json --routes api/routes.json",
+    "http:server": "http-server --proxy http://localhost:3000 -c",
+    "dev": "concurrently --kill-others \"npm run compile:sass\" \"npm run json:server\" \"npm run http:server\""
```

Comment:
👍 
Just an observation, but if you already have `node-sass` installed in project level, then maybe it would be best to also add json-server and `http-server` as dependencies and increase the project portability? 😃